build/checkstyle/tachyon_checks.xml

```diff
@@ -223,6 +223,14 @@
     </module>
     <module name="MissingDeprecated"/>
     <module name="EqualsHashCode"/>
+    <module name="JavadocMethod">
```

Comment:
@HorizonNet , could you add some comment here to explain this rule and the rule below ?
We should keep the checks config file well documented, so we will try to comment other modules too.